### PR TITLE
ocamlformat in Jane-Street only files in typing

### PR DESCRIPTION
**Pull request by goldfirere**
Follow on from #1876.

(This is pointing to #1880, to avoid merge conflicts. First, let's merge #1875, then #1880, then this. I'll do the git-wrangling.)

### CODE REVIEW

**Comment by riaqn** [comment]
While we are at it, could you also add `lambda/translmode.ml*` as well?

**Review by ncik-roberts — APPROVED** [review]
Looks good to me.

File: ocaml/typing/.ocamlformat [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Please make a pull request to change this file.
```
Comment (ncik-roberts) [inline ocaml/typing/.ocamlformat:1]: Please resolve this comment when the PRs you mentioned are merged. Then, you can merge this PR.

Comment (ncik-roberts) [reply ocaml/typing/.ocamlformat:1]: (never mind, it looks like this doesn't block merging. so I'll just resolve this now.)

**Comment by goldfirere** [comment]
#1880 should be merged before this one to make reviewing that PR easier. If #1880 gets sufficiently stalled, I'll just reshuffle this one to go underneath that one.

**Pull request merged**